NameLayerMain/src/vg/civcraft/mc/namelayer/group/groups/BlackList.java

```diff
@@ -1,11 +1,96 @@
 package vg.civcraft.mc.namelayer.group.groups;
 
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
 
-/*
- * Currently wont be implemented.  
- * TTk2 has expressed that he doesnt want this.
- * May come back later to add as an additional feature.
- */
-public class BlackList {
+import vg.civcraft.mc.namelayer.NameLayerPlugin;
+import vg.civcraft.mc.namelayer.group.Group;
 
+public class BlackList {
```

Comment:
Might suffice just to make blacklist functionality optional, if TTK definitely doesn't want it. IMHO it seems pretty useful, solves a lot of the problems I faced with group management back in the day.